src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -447,6 +447,9 @@
   <data name="MultiLinePasteDialog.PrimaryButtonText" xml:space="preserve">
     <value>Paste anyway</value>
   </data>
+  <data name="MultiLinePasteDialog.SecondaryButtonText" xml:space="preserve">
+    <value>Strip line breaks</value>

```

Comment:
```suggestion
    <value>Paste without line breaks</value>
```
Maybe? I feel like we still want "paste" in there so that it's clear that you're pasting and not just modifying the shown text.

@cinnamon-msft Thoughts?

Reply:
I originally had _"Strip line breaks and paste"_ but that felt way too wordy; I do think _"Paste without line breaks"_ is much nicer.

Reply:
Hmm, I seem to get cut-off with the longer text:

![image](https://user-images.githubusercontent.com/651955/110887682-750a1600-8336-11eb-9979-c06117c9057f.png)

Thoughts?

Reply:
Perhaps simply "_Remove line breaks_" would work? I can't really think of a good way to say paste without line breaks without making it too long. "_Paste w/o line breaks_" might work, but that might look weird.